apps/block_scout_web/assets/js/omni.js

```diff
@@ -0,0 +1,163 @@
+// local cache to re-use data
+let txMap = {};
```

Comment:
you probably checked, but there is no way to make some sort of frontend component(s) with some ui framework blockscout is using, we have to do DOM manipulation manually? it seems we are doing some dom manipulation in every loop iteration, which some framework would improve a lot

Reply:
blokscout doesn't use any ui framework, it's server side rendering with bare minimal dynamic js. I am not an ui expert but this is the best solution I could find that can be implemented in the given timeline.

Reply:
perhaps a more explict name here. I wasn't sure what this was for till I read usage